### PR TITLE
BNH-53

### PR DESCRIPTION
**Pull request by TomHanmer29**
Basic restyling of the form for more user-friendliness

### CODE REVIEW

File: web/wwwroot/css/site.css [hunk 1]
```diff
@@ -28,6 +28,7 @@ body {
 
 /* Mark input boxes that gets an error on validation: */
 input.invalid {
+
```
Comment (kwunkiuma) [inline web/wwwroot/css/site.css:31]: Extra newline here 🙂

File: web/wwwroot/js/site.js [hunk 2]
```diff
@@ -50,6 +50,7 @@ function validateForm() {
         {
             // add an "invalid" class to the field:
             tabInputList[tabInputIterator].className += " invalid";
+            tabInputList[tabInputIterator].value =""
```
Comment (kwunkiuma) [inline web/wwwroot/js/site.js:53]: Can we add a space after `=`? 🙂

**Review by kwunkiuma — COMMENTED** [review]
Looks good! Just some minor comments 🙂 

Just to confirm, do the changes address the ticket issue with pressing enter?

**Comment by TomHanmer29** [comment]
As it turns out I'm a muppet and forgot to do what the ticket was actually asking, I just made it easier to tab through. I am changing it now to do what the ticket says :)

**Review by BenRamchandani — APPROVED** [review]

**Pull request merged**